extensions/sql-migration/src/dialog/assessmentResults/sqlDatabasesTree.ts

```diff
@@ -16,32 +16,76 @@ export class SqlDatabaseTree extends AssessmentDialogComponent {
 
 	private createTableComponent(view: azdata.ModelView): azdata.DeclarativeTableComponent {
 
-		const table = view.modelBuilder.declarativeTable().withProperties<azdata.DeclarativeTableProperties>(
```

Comment:
I was referring to the table component, have you evaluated that one?

Reply:
What's the differences between them? They both seemed to be able to do what I need?